features/display_menu_item.feature

```diff
@@ -0,0 +1,24 @@
+Feature: Visitor can see a prodcut on the menu
```

Comment:
Why are there two feature tests added in this PR

Reply:
Display feature was added (and approved) in PR3. This is PR4, where we are adding Add Menu.

Reply:
So you did not branch off from the development branch?

Reply:
We did.